### DISCUSSION TITLE
verifying over-sea residential student

### DISCUSSION THREAD

**Discussion by JiHyeKOO** [community/community#139250] [GitHub Education]
### Hi everyone,

Hi. I'm over-sea residential student of Korea National Open University.
Currently, I live in Japan for my living. 
I updated my student ID and enrollment 4times, every I try, I rejected.

[The school you selected does not appear to have a campus location in your country.]

Could you give me some tips how I can fix this?

**Comment by alx-mp4** [comment]
Hi @JiHyeKOO, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
